packages/alias/alias.loader.mjs

```diff
@@ -40,7 +39,7 @@ export function resolveAliases(specifier, ctx, next) {
 }
 
 export function readConfigFile(filename) {
-	const filepath = path.join(projectRoot.pathname, filename);
+	const filepath = new URL(filename, projectRoot);
```

Comment:
Is this necessary? (constructing a URL is much more expensive than a path)

Reply:
Ah, nevermind. I see the explanation in the issue https://github.com/nodejs-loaders/nodejs-loaders/issues/188 you opened

Reply:
```suggestion
	const filepath = new URL(filename, projectRoot); // URL for cross-compatibility with Windows
```